pkg/render/manager.go

```diff
@@ -431,10 +431,6 @@ func (c *managerComponent) managerEnvVars() []corev1.EnvVar {
 		{Name: "CNX_POLICY_RECOMMENDATION_SUPPORT", Value: "true"},
 		{Name: "ENABLE_MULTI_CLUSTER_MANAGEMENT", Value: strconv.FormatBool(c.cfg.ManagementCluster != nil)},
 		{Name: "ENABLE_KIBANA", Value: strconv.FormatBool(KibanaEnabled(c.cfg.Tenant, c.cfg.Installation))},
-		// The manager supports two states of a product feature being unavailable: the product feature being feature-flagged off,
-		// and the current license not enabling the feature. The compliance flag that we set on the manager container is a feature
-		// flag, which we should set purely based on whether the compliance CR is present, ignoring the license status.
-		{Name: "ENABLE_COMPLIANCE_REPORTS", Value: strconv.FormatBool(c.cfg.Compliance != nil)},
```

Comment:
This flag is currently used by manager UI to show compliance pages and title. UI has made changes to  make user of the Compliance CR existence for this check.